e2e/index.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path')
+const fixtures = require('./fixtures')
+const {Ganache} = require('./ganache')
+const ReferenceCounter = require('./rc')
+const {buildWebDriver, Driver} = require('./webdriver')
+
+const fixtureServer = new fixtures.FixtureServer()
+const rc = new ReferenceCounter(async () => fixtureServer.start(), async () => fixtureServer.stop())
+
+const load = async function load (fixturesDirectory, callback) {
+  await rc.tick(async () => {
```

Comment:
Why do we need the reference counter? Is it to handle multiple simultaneous calls of the fixture server, such that only one instance of the server can ever be running at a time?

Reply:
Yup. We can only have (and only need) the single server on port `12345` so we want to make sure it's only started once and that it's shutdown when the tests end. I'm open to better ways of doing this.